kwamp-client-core/src/main/kotlin/com/laurencegarmstrong/kwamp/client/core/Client.kt

```diff
@@ -47,6 +47,8 @@ class ClientImpl(
     onClose: suspend (message: String) -> Unit = {},
     private val exceptionCatcher: ExceptionCatcher = ExceptionSwallower()
 ) : Client, CoroutineScope by CoroutineScope(Executors.newSingleThreadExecutor().asCoroutineDispatcher()) {
+
+
```

Comment:
So far I don't think any class has whitespace at the top of its variable declarations, so having these two added would make things look inconsistent perhaps :thinking: so maybe we can remove these?